src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Main {
+    public static final Random random = new Random();
```

Comment:
Don't use static methods or static fields in your solution. Static fields can lead to issues in a multi-threaded environment and make testing more difficult. Instead, create an instance of the class within the `main` method and use it.